docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -44,12 +44,10 @@ records the relations between the actions during the above acceptance.
 
 Formally, an **action** is one of the following:
 
-#. a **Create** action on a contract, which records the creation of the contract
```

Comment:
This does sound odd, but is consistent in the Ledger Model. Actions are **on** contracts, and that includes the create node.

Reply:
Sorry, this have been confusing me for a long time along with the inexistent state. We have an infinite set of inexistent contracts in every model then?

Reply:
Yes, that is how the Ledger Model is set up.

Reply:
I reverted this.